### PR TITLE
DICT-453 leeuwenhoek pathology_detail updates

### PR DESCRIPTION
**Pull request by bullenc**
https://gdc-ctds.atlassian.net/browse/DICT-453

### CODE REVIEW

File: src/gdcdictionary/schemas/pathology_detail.yaml [hunk 1]
```diff
@@ -998,12 +1029,19 @@ properties:
         $ref:
           - "_terms_enum.yaml#/not_reported/common"
 
+  tumor_burden:
+    $ref:
+      - "_terms.yaml#/tumor_burden/common"
+    type: number
+    minimum: 0
```
Comment (stilesj-uchicago) [inline src/gdcdictionary/schemas/pathology_detail.yaml:1036]: I did not see a minimum defined in DICT-453

**Review by stilesj-uchicago — APPROVED** [review]
lgtm. It looks like the ticket needs an update to account for a minimum value.

File: src/gdcdictionary/schemas/pathology_detail.yaml (hunk 1 above)
Comment (bullenc) [reply src/gdcdictionary/schemas/pathology_detail.yaml:1036]: Updated ticket, thanks

**Review by wteouchicago — APPROVED** [review]

**Review by wwysoc2 — APPROVED** [review]
👍

**Pull request merged**